index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odin Sign-Up Form</title>
+    <link rel="stylesheet" href="./style.css">
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <div class="main">
+        <div class="left">
+            <div class="forImage">
+                <div class="semi-transparent">
+                    <img src="./Images/odin-lined.png" alt="Odin Logo" height="150px" />
+                    <h1>ODIN</h1>
+                </div>
+            </div>
+        </div>
+        <div class="right">
+            <div class="first">
+                <h3>
+                    This is not a real online service! You know you need something
+                    <br />
+                    like this in your life to help you realize your deepest dreams.
+                    <br />
+                    Sign up <em>now</em> to get started. <br /><br />
+                    You <em>know</em> you want to.
+                </h3>
+            </div>
+            <form action="" method="get">
+                <div class="second">
+                    <h3 style="margin-left: 50px; padding-top: 20px;">Let's do this!</h3>
+                    <div class="form-main-div">
+                        <div class="form-divs">
+                            <div class="form-div">
+                                <label for="firstName">FIRST NAME</label>
+                                <input type="text" name="firstName" id="firstName" placeholder="John" required>
```

Comment:
you can think of adding constraints on input like minLength , pattern etc.

Reply:
okay.